apps/cnquery/cmd/status.go

```diff
@@ -105,7 +105,7 @@ func checkStatus() (Status, error) {
 
 	latestVersion, err := cnquery.GetLatestVersion(httpClient)
 	if err != nil {
-		return s, cli_errors.NewCommandError(errors.Wrap(err, "failed to get latest version"), 1)
+		log.Warn().Err(err).Msg("Failed to get latest cnquery version")
```

Comment:
I think the same code is re-used in `cnspec` so if your message says `Failed to get latest cnquery version`, it would show the same thing when running this from `cnspec`. I think you should keep the original message so it's applicable for both cnquery and cnspec.